src/util/parse.ts

```diff
@@ -1,4 +1,10 @@
 export function parseJSON(json: string): any {
+  // TODO why is this needed?
+  if (json.startsWith('ey')) {
+    const decoded = Buffer.from(json, 'base64').toString('utf8');
+    return JSON.parse(decoded);
+  }
```

Comment:
🤷‍♂️

Reply:
eh wot?

Reply:
yeah dunno I have to dig deeper – somehow this arrives base64 encoded

Reply:
Only thing I can think of is that `[]byte` slices typically get base64-encoded by `json.Marshal` if that's part of the call stack somehow.